nodemon.json

```diff
@@ -3,5 +3,5 @@
   "watch": ["."],
   "ext": ".ts,.js",
   "ignore": ["*.json"],
-  "exec": "ts-node ./index.ts"
+  "exec": "NODE_ENV=dev ts-node ./index.ts"
```

Comment:
not sure it's a good idea to set `NODE_ENV` in `exec` function. it might be executed in prod env too. Also, in some cases you'd want to run `$ NODE_ENV=production nodemon` locally, and this will prevent this process.